src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Base64Url/Base64UrlEncoder.cs

```diff
@@ -251,11 +251,11 @@ public unsafe void EncodeOneOptionallyPadTwo(byte* oneByte, byte* dest, ref byte
 
                 uint i = t0 << 8;
 
-                uint i0 = Unsafe.Add(ref encodingMap, (IntPtr)(i >> 10));
-                uint i1 = Unsafe.Add(ref encodingMap, (IntPtr)((i >> 4) & 0x3F));
+                byte i0 = Unsafe.Add(ref encodingMap, (IntPtr)(i >> 10));
+                byte i1 = Unsafe.Add(ref encodingMap, (IntPtr)((i >> 4) & 0x3F));
 
-                dest[0] = (byte)i0;
-                dest[1] = (byte)i1;
+                dest[0] = i0;
+                dest[1] = i1;
```

Comment:
This could be done as a single ushort write. Not worth it?  Same comment/question for similar cases below.

Reply:
Ah yes, not sure if it's worth, but can update for consistency

> Same comment/question for similar cases below.

Same can be done for char overload, though assume I can keep the 3 bytes writes as is.